components/Makefile

```diff
@@ -161,7 +164,8 @@ $(COMPONENT_DIRS):	$(WS_LOGS) setup FORCE
 
 incorporation:
 	$(eval COMPONENT_REVISION := $(shell git rev-list HEAD --count))
-	$(WS_TOOLS)/userland-incorporator --repository $(PKG_REPO) \
+
```

Comment:
Empty lines in recipes are (or could be) confusing, so I'd prefer to not see it here.  Especially for such short recipes like `incorporation` is.  Thank you.